cmd/sidecar/tracer/handlers/zipkin.go

```diff
@@ -24,31 +24,16 @@ type Zipkin struct {
 	zipkinHost  string        // IP:port of the zipkin service.
 	localHost   string        // IP:port of the sidecare consuming the trace data.
 	sendTimeout time.Duration // Time to wait for the sidecar to respond on send.
-	client      http.Client   // Provides APIs for performing the http send.
+	client      *http.Client  // Provides APIs for performing the http send.
 }
 
 // NewZipkin provides support for publishing traces to zipkin.
 func NewZipkin(zipkinHost string, localHost string, sendTimeout time.Duration) *Zipkin {
-	tr := http.Transport{
-		Proxy: http.ProxyFromEnvironment,
-		DialContext: (&net.Dialer{
-			Timeout:   30 * time.Second,
-			KeepAlive: 30 * time.Second,
-			DualStack: true,
-		}).DialContext,
-		MaxIdleConns:          2,
-		IdleConnTimeout:       90 * time.Second,
-		TLSHandshakeTimeout:   10 * time.Second,
-		ExpectContinueTimeout: 1 * time.Second,
-	}
-
 	z := Zipkin{
 		zipkinHost:  zipkinHost,
 		localHost:   localHost,
 		sendTimeout: sendTimeout,
-		client: http.Client{
-			Transport: &tr,
-		},
+		client:      web.NewHTTPClient(),
```

Comment:
I don't see where this function is being implemented under web?

Reply:
Yes, platform/web/client.go